recipes/xcengine/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "xcengine" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xcube-dev/xcengine/archive/v{{ version }}.tar.gz
+  sha256: bdb335c578d1a88ce2e424058f67d055c487c762f145938492264aafd64cc520
+
+build:
+  entry_points:
+    - xcetool = xcengine.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=40.9.0
+    - pip
+  run:
+    - python >={{ python_min }}
+    - click
+    - docker-py
+    - ipython
+    - nbconvert
+    - nbformat
+    - pystac
+    - pyyaml
+    - xarray
+    - xcube
+
+test:
+  imports:
+    - xcengine
+  commands:
+    # "pip check" omitted, since it expects a package named xcube-core
```

Comment:
It would be nice to get this fixed upstream, otherwise we force any packages depending on this and xcube to avoid an important test.

Reply:
Thanks for the suggestion -- I would like to fix this, but I don't know how. We don't have the option to rename the xcube-core PyPI package (the PyPI name "xcube" is taken by another project), and I couldn't find any way to inform "pip check" that the import package xcube is provided by the distribution package xcube-core. If you have any advice on how to solve this, I'd be very grateful!